package.json

```diff
@@ -74,8 +83,6 @@
     "eslint-plugin-prettier": "^5.1.3",
     "jest": "^29.7.0",
     "prettier": "^3.2.5",
-    "react": "^18.2.0",
```

Comment:
@boorad One question ... was this change to move the deps to main deps intended? if you can ths mean conflicts with otehr versions like current 0.74.x?